### DISCUSSION TITLE
How do I get someone off my account? Where do I start?

### DISCUSSION THREAD

**Discussion by lydretz1982** [community/community#76817] [Apps, API and Webhooks]
### Select Topic Area

Bug

### Body

Someone has been in my account doing all this GitHub stuff and changing settings. How do I get my account back? I would like to learn some of this coding stuff please

**Comment by ghostinhershell** [comment]
Hi @lydretz1982, 

Thanks so much for reaching out here. To make sure that I'm understanding this correctly, do you believe that your account has been hacked? If so, the best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them.
